composer.json

```diff
@@ -17,7 +17,7 @@
   "require": {
     "composer/installers": "^1.3.0",
     "pantheon-systems/quicksilver-pushback": "^1.0",
-    "pantheon-systems/wordpress-composer": "^4.8.2",
+    "pantheon-systems/wordpress-composer": "^5.0.0",
```

Comment:
looks like it's still at 5.0.0?

Reply:
The way Composer requirements get expressed is that this change means "at least 5.0.0, until you get to 6.0.0" - so we wouldn't need to update `composer.json` with any further changes until WP6 comes out.

Reply:
Composer's treatment of versions and the `^` or `~` or other characters is documented here, btw - I'm working on internalizing this today: https://getcomposer.org/doc/articles/versions.md